workbench/runtime.py

```diff
@@ -239,13 +241,22 @@ class WorkbenchRuntime(Runtime):
     def __init__(self, user_id=None):
         #  TODO: Add params for user, runtime, etc. to service initialization
         #  Move to stevedor
-        services={
+        services = {
             'field-data': KvsFieldData(WORKBENCH_KVS),
             'user': WorkBenchUserService(user_id),
         }
 
-        if hasattr(xblock.reference.plugins, 'FSService'):
-            services['fs'] = xblock.reference.plugins.FSService()
+        # Load additional services defined by Django settings
+        # This is useful for instances of workbench used to develop
+        # XBlocks that require services that may be too specific
+        # to include in the default workbench configuration.
+        for service_name, service_path in settings.WORKBENCH.get('services', {}).iteritems():
+            if service_name in services:
+                log.error('Service %s has already been defined.', service_name)
```

Comment:
optional / suggestion: removing this would allow authors to work against custom implementations of base services, which I could imagine coming in handy.